content/documentation/distributed-tracing/index.adoc

```diff
@@ -112,4 +112,4 @@ If you are working in a development environment, you can see an error as the ima
 
 image::certificateErr.png[Certificate Error]
 
-This is because your certificate is not valid, so you need to accept the certificate, open the tracing service in a new tab and accept it, after that you need to refresh the Kiali website.
\ No newline at end of file
+This is because your certificate is not valid, so you need to accept the certificate, open the tracing service in a new tab and accept it, after that you need to refresh the Kiali website.
```

Comment:
```suggestion
This is because your certificate is not valid, so you need to accept the certificate.  Open the tracing service in a new tab and accept the certificate.  Then you need to refresh the Kiali website.
```